commit-boost/config.toml

```
@@ -0,0 +1,8 @@
+chain = "Mainnet"
+
+[pbs]
+port = 18550
```

```suggestion
port = 18550
host="0.0.0.0"
```

default is 127.0.0.1 which does not work in containers